### PR TITLE
Implement GetCursorPosition() plug

### PR DESCRIPTION
**Pull request by Kiirx**

### CODE REVIEW

File: source/Cosmos.System2_Plugs/System/ConsoleImpl.cs [hunk 1]
```diff
@@ -545,12 +544,22 @@ public static void SetBufferSize(int width, int height)
             throw new NotImplementedException("Not implemented: SetBufferSize");
         }
 
+        System.ConsolePal::System.ValueTuple`2[System.Int32,System.Int32] GetCursorPosition();
```
Comment (quajak) [inline source/Cosmos.System2_Plugs/System/ConsoleImpl.cs:547]: Why is this required?

**Review by MishaTY — APPROVED** [review]

**Review by quajak — CHANGES_REQUESTED** [review]
Thanks for the PR!
It looks good but could you please add a test to https://github.com/CosmosOS/Cosmos/blob/48239ffa0501098e235d1de8a1f97422c68d9275/Tests/Kernels/ConsoleTest/Kernel.cs#L155 This ensures that it wont break randomly in the future.

File: source/Cosmos.System2_Plugs/System/ConsoleImpl.cs (hunk 1 above)
Comment (Kiirx) [reply source/Cosmos.System2_Plugs/System/ConsoleImpl.cs:547]: I removed it normally

Comment (Kiirx) [reply source/Cosmos.System2_Plugs/System/ConsoleImpl.cs:547]: oh I forgot
sorry

**Comment by Kiirx** [comment]
> Thanks for the PR! It looks good but could you please add a test to
> 
> https://github.com/CosmosOS/Cosmos/blob/48239ffa0501098e235d1de8a1f97422c68d9275/Tests/Kernels/ConsoleTest/Kernel.cs#L155
> 
> This ensures that it wont break randomly in the future.

Done

**Review by quajak — APPROVED** [review]
